stories/MewInput/MewInput.stories.js

```diff
@@ -47,6 +47,9 @@ export const MEWInput = () => ({
     isSearch: {
       default: boolean('is-search', false)
     },
+    type: {
+      default: text('type', 'text')
```

Comment:
hmm i think this one we would want to give them the options instead, you can use optionsKnob for this! (you can see the example in Toast.stories.js)